File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/AttendanceAct/dataModel.java
package com.example.adminapp.AttendanceAct;

public class dataModel {
    private int pImg;
    private String pName,pRollNo;

    public dataModel() {} //no-args constructor

    public dataModel(int pImg, String pName, String pRollNo) {
        this.pImg = pImg;
        this.pName = pName;
        this.pRollNo = pRollNo;
    }

    public int getpImg() {
        return pImg;
    }

    public void setpImg(int pImg) {
        this.pImg = pImg;
    }

    public String getpName() {
        return pName;
    }

    public void setpName(String pName) {
        this.pName = pName;
    }

    public String getpRollNo() {
        return pRollNo;
    }

    public void setpRollNo(String pRollNo) {
        this.pRollNo = pRollNo;
    }
}

File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/AttendanceAct/dataModel.kt
package com.example.adminapp.AttendanceAct

class dataModel {
    private var pImg = 0
    private var pName: String? = null
    private var pRollNo: String? = null

    constructor() //no-args constructor
    constructor(pImg: Int, pName: String?, pRollNo: String?) {
        this.pImg = pImg
        this.pName = pName
        this.pRollNo = pRollNo
    }

    fun getpImg(): Int {
        return pImg
    }

    fun setpImg(pImg: Int) {
        this.pImg = pImg
    }

    fun getpName(): String? {
        return pName
    }

    fun setpName(pName: String?) {
        this.pName = pName
    }

    fun getpRollNo(): String? {
        return pRollNo
    }

    fun setpRollNo(pRollNo: String?) {
        this.pRollNo = pRollNo
    }
}
